src/SuperBlog/Controller/ArticleController.php

```diff
@@ -23,8 +26,15 @@ public function __construct(ArticleRepository $repository, Twig_Environment $twi
         $this->twig = $twig;
     }
 
-    public function show($id)
+    /**
+     * @param $id
+     * @throws LoaderError
+     * @throws RuntimeError
+     * @throws SyntaxError
+     */
```

Comment:
The phpdoc is not necessary because this is a controller, no human will call the method.